app/src/main/java/com/rahobbs/todo/helpers/TodoItem.java

```diff
@@ -0,0 +1,88 @@
+package com.rahobbs.todo.helpers;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/1/16.
+ * Model for a single to-do item
+ */
+public class TodoItem {
+    private String mTitle;
+    private String mDetails;
+    private Date mDate;
+    private UUID mID;
+    private boolean completed;
+    private String mParents;
+    private String mChildren;
+
+
+    public TodoItem() {
+        mID = UUID.randomUUID();
+        mDate = new Date();
+    }
+
+    public TodoItem(UUID id) {
+        mID = id;
+        mDate = new Date();
+    }
+
+    public boolean isCompleted() {
```

Comment:
I know Android Studio has a super easy way to supply getters and setters for every member variable in a class, but I think that's bad habit and leads to some bloat. 

I recommend thinking of your POJOs as APIs. So every method on your POJO is something you want someone to be able to do to this object. But in the case of auto generating, you're opening up methods that you might not want to be exposed.